missioncontrol/home/migrations/0021_s3files_20190227_0220.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 2.1.4 on 2019-02-27 03:37
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import home.models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('home', '0020_merge_20190124_0028'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='S3File',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('cid', models.CharField(max_length=128, unique=True)),
+                ('bucket', models.CharField(max_length=64)),
+                ('what', models.TextField(default='default')),
+                ('created', models.DateTimeField(default=django.utils.timezone.now)),
+                ('where', models.TextField(default='nowhere')),
```

Comment:
I don't think `nowhere` is a useful default. How about making it nullable?
Ideally this would be a FQDN, or a hwid.

Reply:
This was a migration only - the field itself has no default and is not nullable on the model.